addons/l10n_ug/data/template/account.tax-ug.csv

```diff
@@ -0,0 +1,81 @@
+,,,,,,,"refund","100","tax",,,
+"sale_exempt","Tax Exempt","Supplies exempts of tax.","0","sale",,"consumption_taxes","invoice","100","base",,"+0% Exempt Sales Base","True"
+,,,,,,,"invoice","100","tax",,,
+,,,,,,,"refund","100","base",,"-0% Exempt Sales Base",
+,,,,,,,"refund","100","tax",,,
+"sale_vat_18","18%","VAT on all supplies made by taxable persons.","18","sale",,"consumption_taxes","invoice","100","base",,"+18% VAT Sales Base","True"
+,,,,,,,"invoice","100","tax","411722","+18% VAT Sales Tax",
+,,,,,,,"refund","100","base",,"-18% VAT Sales Base",
+,,,,,,,"refund","100","tax","411722","-18% VAT Sales Tax",
+"purchase_vat_18","18%","VAT on all supplies made by taxable persons.","18","purchase",,"consumption_taxes","invoice","100","base",,"+18% VAT Purchases Base","True"
+,,,,,,,"invoice","100","tax","352804","+18% VAT Purchases Tax",
+,,,,,,,"refund","100","base",,"-18% VAT Purchases Base",
+,,,,,,,"refund","100","tax","352804","-18% VAT Purchases Tax",
+"sale_deemed_vat_18","18% Deemed","Deemed VAT on all supplies made by taxable persons.","18","sale",,"consumption_taxes","invoice","100","base",,"+18% Deemed VAT Sales Base","False"
+,,,,,,,"invoice","100","tax","411722","+18% Deemed VAT Sales Tax",
+,,,,,,,"invoice","-100","tax","411722",,
+,,,,,,,"refund","100","base",,"-18% Deemed VAT Sales Base",
+,,,,,,,"refund","100","tax","411722","-18% Deemed VAT Sales Tax",
+,,,,,,,"refund","-100","tax","411722",,
+"purchase_deemed_vat_18","18% Deemed","Deemed VAT on all supplies made by taxable persons.","18","purchase",,"consumption_taxes","invoice","100","base",,"+18% Deemed VAT Purchases Base","False"
+,,,,,,,"invoice","100","tax","352804","+18% Deemed VAT Purchases Tax",
+,,,,,,,"invoice","-100","tax","352804",,
+,,,,,,,"refund","100","base",,"-18% Deemed VAT Purchases Base",
+,,,,,,,"refund","100","tax","352804","-18% Deemed VAT Purchases Tax",
+,,,,,,,"refund","-100","tax","352804",,
+"sale_capital_goods_18","18% CG","Tax on all capital goods (business assets) sold.","18","sale",consu,"consumption_taxes","invoice","100","base",,"+18% CG Sales Base","True"
+,,,,,,,"invoice","100","tax","411722","+18% CG Sales Tax",
+,,,,,,,"refund","100","base",,"-18% CG Sales Base",
+,,,,,,,"refund","100","tax","411722","-18% CG Sales Tax",
+"purchase_capital_goods_18","18% CG","Tax on all capital goods (business assets) purchased.","18","purchase",consu,"consumption_taxes","invoice","100","base",,"+18% CG Purchases Base","True"
+,,,,,,,"invoice","100","tax","352804","+18% CG Purchases Tax",
+,,,,,,,"refund","100","base",,"-18% CG Purchases Base",
+,,,,,,,"refund","100","tax","352804","-18% CG Purchases Tax",
+"sale_deemed_capital_goods_18","18% CG Deemed","Deemed tax on all capital goods (business assets) sold.","18","sale",consu,"consumption_taxes","invoice","100","base",,"+18% Deemed CG Sales Base","False"
+,,,,,,,"invoice","100","tax","411722","+18% Deemed CG Sales Tax",
+,,,,,,,"invoice","-100","tax","411722",,
```

Comment:
How is this tax reverse charge?

Reply:
Sorry, I don't understand what you mean with that question. Could you rephrase ?